### ISSUE TITLE
Minor Comments

### ISSUE THREAD

**Issue by mufeili** [zhjwy9343/MVP4ModelExample#4]
For README:

1. "The author's codes of implementation is in here". I think we can just say "For the original implementation, see [here](link)". Note that code is uncountable, so no "s".
2. I don't think "implementor" is the right term to use. Maybe we can replace it with "author" or "contributor". 
3. "This example was implemented by zhjwy9343 during his *** work at the AWS Shanghai AI Lab. Note: Please replace the implementor with your identity and *** with your real status when implement this example." I don't think identity and institution matter here.

For main.py:
1. Remove L269: # Test with mini batch after all epoch
2. Replace `model.forward(xxx)` with `model(xxx)`. The base class `torch.nn.Module` invokes `forward` in `__call__` for a syntax sugar and this has been the standard practice.
3. For the evaluation on the validation set, you need to call `model.eval()` to disable dropout, fix batchnorm and add `with torch.no_grad()` to disable gradient.
4. No need to compute loss on the test set.
